src/components/HappiGraph/happi-graph.component.tsx

```diff
@@ -52,15 +55,41 @@ interface State {
   allGroup: any;
   isFullscreen: boolean;
   printMode: boolean;
+
+  addLinks: any;
+  addNodes: any; 
+  centerGraph: any;
+  customZoomIn: any; 
+  customZoomOut: any; 
+  initCenterGraph: any;
+  graphType: number;
```

Comment:
why were these added?

Reply:
It was the way I tried to inlcude the addNodes/Links methods, ignore that I've changed it now and removed this